backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -39,11 +38,6 @@ public abstract class BaseTestInfo extends AuditedEntity implements Organization
   @JoinColumn(name = "specimen_type_id")
   private SpecimenType specimenType;
 
-  @Column(nullable = true)
-  @Type(type = "pg_enum")
-  @Enumerated(EnumType.STRING)
-  private TestResult result;
```

Comment:
Should we be deleting this field or just like deprecating it?

Reply:
I think just deprecating it - the `result` column is still on the schema definition, and I'm not sure how Java/liquibase will response if the schema definition doesn't exactly match the Spring models.

Reply:
Ok yeah, I didn't know if it would screw something up! Just added it back and slapped a `@Deprecated` on it.